src/config/unit_test.rs

```diff
@@ -9,7 +9,9 @@ use indexmap::IndexMap;
 use std::{collections::HashMap, path::PathBuf};
 
 pub async fn build_unit_tests_main(path: PathBuf) -> Result<Vec<UnitTest>, Vec<String>> {
-    let config = super::loading::load_builder_from_paths(&[path])?;
+    let mut warnings = Vec::new();
+    let config = super::loading::load_builder_from_paths(&[path], &mut warnings)?;
+    super::handle_warnings(warnings);
```

Comment:
It seems like we call `super::handle_warnings` every time we call the config builder? Could we move that logic down?